README.md

```diff
@@ -196,11 +200,19 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/thititongumpun"><img src="https://avatars.githubusercontent.com/u/55313215?v=4?s=100" width="100px;" alt="thititongumpun"/><br /><sub><b>thititongumpun</b></sub></a><br /><a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Athititongumpun" title="Bug reports">🐛</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=thititongumpun" title="Code">💻</a> <a href="#data-thititongumpun" title="Data">🔣</a> <a href="#design-thititongumpun" title="Design">🎨</a> <a href="#infra-thititongumpun" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
+<<<<<<< HEAD
+<<<<<<< HEAD
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Obasoro"><img src="https://avatars.githubusercontent.com/u/29310552?v=4?s=100" width="100px;" alt="Obasoro"/><br /><sub><b>Obasoro</b></sub></a><br /><a href="#a11y-Obasoro" title="Accessibility">️️️️♿️</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3AObasoro" title="Bug reports">🐛</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=Obasoro" title="Code">💻</a> <a href="#content-Obasoro" title="Content">🖋</a> <a href="#data-Obasoro" title="Data">🔣</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=Obasoro" title="Documentation">📖</a> <a href="#example-Obasoro" title="Examples">💡</a> <a href="#ideas-Obasoro" title="Ideas, Planning, & Feedback">🤔</a> <a href="#infra-Obasoro" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#mentoring-Obasoro" title="Mentoring">🧑‍🏫</a> <a href="#research-Obasoro" title="Research">🔬</a> <a href="https://github.com/TejasQ/opensauced-guestbook/pulls?q=is%3Apr+reviewed-by%3AObasoro" title="Reviewed Pull Requests">👀</a> <a href="#security-Obasoro" title="Security">🛡️</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=Obasoro" title="Tests">⚠️</a> <a href="#tool-Obasoro" title="Tools">🔧</a> <a href="#userTesting-Obasoro" title="User Testing">📓</a></td>
+=======
```

Comment:
This should be generated once after you run the `npm run contributors:generate`.
But there is a duplication below.

Delete them both and run `npm run contributors:generate` after you added yourself in `.all-contributorsrc`.